fem/fespace.hpp

```diff
@@ -660,76 +781,261 @@ class FiniteElementSpace
 
    int GetBdrAttribute(int i) const { return mesh->GetBdrAttribute(i); }
 
-   /// Returns indices of degrees of freedom of element 'elem'.
+   /// @anchor getdof @name Local DoF Access Members
+   /// These member functions produce arrays of local degree of freedom
+   /// indices, see @ref ldof. If @b vdim == 1 these indices can be used to
+   /// access entries in GridFunction, LinearForm, and BilinearForm objects.
+   /// If @b vdim != 1 the corresponding @ref getvdof "Get*VDofs" methods
+   /// should be used instead or one of the @ref dof2vdof "DofToVDof" methods
+   /// could be used to produce the appropriate offsets from these local dofs.
+   ///@{
+
+   /// @brief Returns indices of degrees of freedom of element 'elem'.
+   /// The returned indices are offsets into an @ref ldof vector. See also
+   /// GetElementVDofs().
+   ///
+   /// @note In many case the returned DofTransformation object will be NULL.
```

Comment:
Yes, there is. I sometimes hesitate to mention such things because you never know what might be added in the future and perhaps these cases will expand. However, I'll try to add something with suitable caveats.

Reply:
Let me do this: I'll expand the header documentation for `FiniteElementSpace` where `DofTransformation` is mentioned. I'll also expand the `note` sections of the methods that return `DofTransformation` objects. If this seems to be overkill we can revert one or the other of these.